src/Ocelot.Administration/Ocelot.Administration.csproj

```diff
@@ -27,11 +27,11 @@
     <ProjectReference Include="..\Ocelot\Ocelot.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.66">
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.164">

```

Comment:
Still no stable version available of this package?

Reply:
Unfortunately not.